trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0, 4)
+  end
+end
```

Comment:
- [ ] Also, in the `TrimmerDecorator` class, you needed to Implement a method `correct_name` that makes sure that the output of `@nameable.correct_name` has a maximum of 10 characters. You did a great job creating the method. However, currently, your `correct_name` method trims the word to a maximum of 4 characters instead of 10 so kindly fix that as well :wink: 
- Your Current Output
![Screenshot from 2023-08-30 20-36-44](https://github.com/CesarHerr/OOP_school_library/assets/102699958/8f33316d-9f7b-4a1c-9554-40dbf0ecd39e)

Reply:
@NahnahAJ  You are right! I forgot to change that, I'll fix it. Thanks.